python/doc/examples/numerical_methods/estimate_integral_Clenshaw_Curtis.ipynb

```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Estimate an integral"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this basic example we are going to evaluate an integral of the form.\n",
+    "\n",
+    "$$I_f = \\int_{a}^{b}\\, f(x)\\mathrm{d}{x}$$\n",
+    "\n",
+    "with the iterated quadrature algorithm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function\n",
+    "import openturns as ot\n",
+    "import math as m"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define the integrand and the bounds\n",
```

Comment:
This comment, and all comments in the cells below, could be put in independent markdown cells in order to make the notebook less script-like.